.vscode/extensions.json

```diff
@@ -1,6 +1,6 @@
 {
   "recommendations": [
-    "ms-vscode.csharp",
+    "ms-dotnettools.csharp",
```

Comment:
What does this affect? Just a nit you hit while doing the other changes?

Reply:
Yup. This in unrelated. This is the new name for the vscode C# extension.